src/request.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-console */
+'use strict';
+
+const http = require('http');
```

Comment:
The task is to create a server, not to make a request. You should use http.createServer method instead of http.get.